Objects/codeobject.c

```diff
@@ -1427,6 +1428,7 @@ code_dealloc(PyCodeObject *co)
         PyMem_Free(co->co_quickened);
         _Py_QuickenedCount--;
     }
+    Py_XDECREF(co->co_hydra_context);
```

Comment:
I wonder if this should move up so it's right after Py_XDECREF(co->co_exceptiontable)?